src/components/views/elements/Dropdown.js

```diff
@@ -115,7 +115,7 @@ export default class Dropdown extends React.Component {
 
     componentWillReceiveProps(nextProps) {
         this._reindexChildren(nextProps.children);
-        const firstChild = React.Children.toArray(nextProps.children)[0];
+        const firstChild = nextProps.children[0];
```

Comment:
Ah right. I think there's still a problem though as it would be valid to pass only a single element to a dropdown, in which case this.props.children would not be an array?